app/pages/admin/project-status.jsx

```diff
@@ -85,17 +86,21 @@ class ProjectStatus extends Component {
     this.setState({ dialogIsOpen: false });
   }
 
-  handleDialogSuccess(workflow) {
+  handleDialogSuccess() {
+    const defaultWorkflow = this.state.workflows.filter(workflow => workflow.id === this.state.defaultWorkflowId);
```

Comment:
I wanted to call this variable `workflow`, but that raised a linter error (`'workflow' is already declared in the upper scope`), so I went with defaultWorkflow.

Reply:
Good use of `array.filter()`, however, we can sidestep storing the default workflow id in the component state since it's already stored in the project resource:
`const defaultWorkflow = this.state.workflows.filter(workflow => workflow.id === this.state.project.configuration.default_workflow);`